content/pt-br/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -0,0 +1,125 @@
+
+Kubespray fornece as seguintes utilidades para ajudar a provisionar seu ambiente:
+
+* Scripts do [Terraform](https://www.terraform.io/) para os seguintes provedores de nuvem:
+  * [AWS](https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/aws)
+  * [OpenStack](https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/openstack)
+  * [Equinix Metal](https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/metal)
+
+### (2/5) Componha um arquivo de inventário
+
+Depois de provisionar seus servidores, crie um [arquivo de inventário para o Ansible](https://docs.ansible.com/ansible/latest/network/getting_started/first_inventory.html). Você pode fazer isso manualmente ou por meio de um script de inventário dinâmico. Para mais informações, consulte "[Criando seu próprio inventário](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#building-your-own-inventory)".
+
+### (3/5) Planeje o deployment do seu cluster
+
+O Kubespray oferece a habilidade de customizar vários aspectos do deployment:
+
+* Escolha do modo de deployment: kubeadm ou non-kubeadm
+* Plugins CNI (rede)
+* Configuração de DNS
+* Escolha da camada de gerenciamento: nativo/binário ou conteinerizado
+* Versões de componentes
+* Reflexões de rota Calico
+* Opções de agente de execução do componente
+  * {{< glossary_tooltip term_id="docker" >}}
+  * {{< glossary_tooltip term_id="containerd" >}}
+  * {{< glossary_tooltip term_id="cri-o" >}}
+* Métodos de geração de certificados
+
+As customizações do Kubespray podem ser feitas em um [arquivo variável](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html). Se você estiver começando com o Kubespray, considere usar os padrões do Kubespray para implantar seu cluster e explorar o Kubernetes.
+
+### (4/5) Deploy um Cluster
+
+Em seguida, deploy seu cluster:
+
+Deployment do cluster usando [ansible-playbook](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#starting-custom-deployment).
+
+```shell
+ansible-playbook -i your/inventory/inventory.ini cluster.yml -b -v \
+  --private-key=~/.ssh/private_key
+```
+
+Grandes deployments (mais de 100 nós) podem exigir [adjustes específicos](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/large-deployments.md) para obter melhores resultados.
+
+### (5/5) Verifique o deployment
+
+O Kubespray fornece uma maneira de verificar a conectividade entre pods e a resolução de DNS com o [Netchecker](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/netcheck.md). O Netchecker garante que os pods de agentes do netchecker possam resolver solicitações de DNS e pingar cada um dentro do namespace padrão. Esses pods imitam um comportamento semelhante ao restante das cargas de trabalho e servem como indicadores de integridade do cluster.
```

Comment:
I think the use of "deployment" should be rethought in these sentences. I don't think any of those refer to the actual Kubernetes deployment object. Therefore, they should probably be translated.